### PR TITLE
Jamescarlyle webapp obscuroscan

### PR DESCRIPTION
**Pull request by jamescarlyle**
### Why is this change needed?
Change deployment of Obscuroscan to Azure web app

- Provide a description and a link to the underlying ticket
https://github.com/obscuronet/go-obscuro/issues/848

### What changes were made as part of this PR:
Updated deployment script based on Azure Docker deployment.

### :rotating_light: Definition of done :rotating_light:
- [NA] Unit tests added to cover new or changed functionality 
- [NA] Docs pages updated to cover new or changed functionality
- [NA] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

**Comment by tudor-malene** [comment]
@otherview , I think you're best placed to review this PR

File: .github/workflows/manual-deploy-obscuroscan.yml [hunk 1]
```diff
@@ -1,71 +1,38 @@
-# Deploys Obscuroscan on Azure for Testnet
-# Builds the Obscuroscan network image, pushes the image to dockerhub and starts the Obscuroscan on azure
-#
-# Obscuroscan is a docker container that runs the obscuroscan package
-# It exposes the following ports:
-# HTTP:       80,
-#
-# Exposes the following addresses: (only accessible internally)
-#  testnet-obscuroscan.uksouth.azurecontainer.io
-
```
[the remark this answers is not in the document]
Comment (jamescarlyle) [reply .github/workflows/manual-deploy-obscuroscan.yml:10]: yep

File: .github/workflows/manual-deploy-obscuroscan.yml [hunk 2]
```diff
@@ -1,71 +1,38 @@
...
 name: '[M] Deploy Obscuroscan'
 
-on:
-  workflow_dispatch:
+on: push
```
[the remark this answers is not in the document]
Comment (jamescarlyle) [reply .github/workflows/manual-deploy-obscuroscan.yml:3]: yep

File: .github/workflows/manual-deploy-obscuroscan.yml [hunk 3]
```diff
@@ -1,71 +1,38 @@
...
+  AZURE_WEBAPP_NAME: testnet-obscuroscan
+  CONTAINER_REGISTRY: ${{ secrets.REGISTRY_LOGIN_SERVER }}  # set secret with Container Registry URL, example : xyz.azurecr.io 
+  AZURE_RESOURCE_GROUP: testnet   # set this to your Azure Resource group's name - Needed only if you are provisioning the app in the workflow
+  AZURE_APP_PLAN: obscuroscan-plan  # set this to your App service plan's name - Needed only if you are provisioning the app in the workflow
+  
 jobs:
-  build-and-deploy:
+  build-and-deploy-to-dev:
     runs-on: ubuntu-latest
+    environment: dev
```
[the remark this answers is not in the document]
Comment (jamescarlyle) [reply .github/workflows/manual-deploy-obscuroscan.yml:14]: Yeah, not sure why this got included - I took inspiration from a template, but I see that we don't specify GH environments anywhere else, so I've taken it out and reverted the command to "build-and-deploy"

**Review by otherview — APPROVED** [review]

**Pull request merged**